[2021-12-01 17:51:08,928] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.accurate scheduled__2021-11-21T00:00:00+00:00 [queued]>
[2021-12-01 17:51:08,933] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.accurate scheduled__2021-11-21T00:00:00+00:00 [queued]>
[2021-12-01 17:51:08,933] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 17:51:08,933] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-01 17:51:08,933] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 17:51:08,940] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): accurate> on 2021-11-21 00:00:00+00:00
[2021-12-01 17:51:08,942] {standard_task_runner.py:52} INFO - Started process 54174 to run task
[2021-12-01 17:51:08,946] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'accurate', 'scheduled__2021-11-21T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/var/folders/xt/_j47lzt57cj88vmq2807qx_00000gp/T/tmpj4qdb5g7', '--error-file', '/var/folders/xt/_j47lzt57cj88vmq2807qx_00000gp/T/tmprpjwsqta']
[2021-12-01 17:51:08,948] {standard_task_runner.py:77} INFO - Job 24: Subtask accurate
[2021-12-01 17:51:08,979] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.accurate scheduled__2021-11-21T00:00:00+00:00 [running]> on host zipcodes-macbook-pro-6.local
[2021-12-01 17:51:09,002] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2021-11-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-21T00:00:00+00:00
[2021-12-01 17:51:09,003] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/xt/_j47lzt57cj88vmq2807qx_00000gp/T
[2021-12-01 17:51:09,003] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2021-12-01 17:51:09,010] {subprocess.py:85} INFO - Output:
[2021-12-01 17:51:09,017] {subprocess.py:89} INFO - accurate
[2021-12-01 17:51:09,018] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-01 17:51:09,033] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=accurate, execution_date=20211121T000000, start_date=20211201T225108, end_date=20211201T225109
[2021-12-01 17:51:09,080] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-01 17:51:09,096] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
